### ISSUE TITLE
Invalid yaw/pitch falses

### ISSUE THREAD

**Issue by iFlamingoz** [MWHunter/Grim#472]
**Describe the bug**
Invalid pitch/yaw false positives whilst turning with a 1.8 client/server.

**To Reproduce**
https://youtu.be/0yTUuhNOVLY

**Server Version:** 1.8.8

**Grim Version (Git commit or the version in the jar's name)** 101909ed83cae6fe948c102182a71709b39113cd

**Relevant plugins**
Or all plugins if you are unsure

**Comment by MWHunter** [comment]
Can you reproduce this with vanilla 1.8?  I can't flag this check on vanilla or lunar.

**Comment by MWHunter** [comment]
I added debug with https://github.com/MWHunter/Grim/commit/60328df79978cd75e713783594ba29151d01e5d0

**Comment by iFlamingoz** [comment]
Yes, I can reproduce this with vanilla.
With debug:
![image](https://user-images.githubusercontent.com/35672743/167278671-bd1c228b-2e15-4437-9fe3-2f6b0f015a26.png)

**Comment by MWHunter** [comment]
What FPS do you have?

**Comment by iFlamingoz** [comment]
> What FPS do you have?

I have around 500-1500 fps and an average of around 800

**Comment by MWHunter** [comment]
yeah... high fps is falsing the check as floats are getting rounded 75 times a tick.  I have no clue how to fix this other than deleting the check as the offset 0.00024 is what clients do when they mess up the player's rotation.

**Comment by iFlamingoz** [comment]
> yeah... high fps is falsing the check as floats are getting rounded 75 times a tick. I have no clue how to fix this other than deleting the check as the offset 0.00024 is what clients do when they mess up the player's rotation.

You could either delete it or let people choose to enable/disable it in the config.
It doesn't seem like a good check if players can flag it just by having high fps.

**Comment by MWHunter** [comment]
I'll just delete it and hope GCD catches clients messing with this stuff.
